File: app/src/main/java/com/parabola/newtone/ui/dialog/IsaacNewtoneDialog.java
package com.parabola.newtone.ui.dialog;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.parabola.newtone.R;

import butterknife.ButterKnife;
import butterknife.OnClick;

public final class IsaacNewtoneDialog extends BaseDialogFragment {

    @NonNull
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View layout = inflater.inflate(R.layout.dialog_isaac_newtone, container, false);
        ButterKnife.bind(this, layout);

        return layout;
    }

    @OnClick(R.id.cancel)
    public void onClickCancel() {
        dismiss();
    }
}

File: app/src/main/java/com/parabola/newtone/ui/fragment/settings/dialog/IsaacNewtoneDialog.kt
package com.parabola.newtone.ui.fragment.settings.dialog;

import android.app.Dialog;
import android.os.Bundle;
import android.widget.ImageView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.widget.AppCompatImageView;

import com.google.android.material.dialog.MaterialAlertDialogBuilder;
import com.parabola.newtone.R;

import moxy.MvpAppCompatDialogFragment;

import static androidx.core.content.ContextCompat.getDrawable;

public final class IsaacNewtoneDialog extends MvpAppCompatDialogFragment {

    private ImageView newtoneImage;

    @NonNull
    @Override
    public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
        newtoneImage = new AppCompatImageView(requireContext());
        newtoneImage.setImageDrawable(getDrawable(requireContext(), R.drawable.isaac_newtone));

        return new MaterialAlertDialogBuilder(requireContext())
                .setTitle(R.string.app_name)
                .setNegativeButton(R.string.dialog_cancel, null)
                .setView(newtoneImage)
                .create();
    }

}
